live-examples/js-examples/bigint/bigint-asintn.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">const max = 2n ** (64n - 1n) - 1n;
+
+function check64bit(number) {
+  (number > max) ? console.log("Number doesn't fit in signed 64-bit integer!") : console.log(BigInt.asIntN(64, number));
```

Comment:
This line and the equivalent one for `asUintN()` is too long: as a rule of thumb I'd like them to be at most 1000px wide, so the user can see the whole thing without having to scroll. It's worse in this case too because the API being demonstrated isn't shown until character 99 of the line so it's very easy to miss.

I'd suggest formatting this line like:

```js
  (number > max) ?
    console.log("Number doesn't fit in signed 64-bit integer!") :
    console.log(BigInt.asIntN(64, number));
```

I think this also makes the branching logic of the line clearer.

Reply:
This makes sense to me. I think I need to remember to have the local test interactive-examples not in a large window. Did we think about doing something about this? For example, setting a fixed width for testing purposes locally? Just a thought, though.

Reply:
We do have an issue for this: https://github.com/mdn/bob/issues/166 but I don't know when it might be fixed.